### ISSUE TITLE
finetuning with caption_cn_large

### ISSUE THREAD

**Issue by dinglei8908** [OFA-Sys/OFA#272]
I wonder the mini batch size when finetuning with caption_cn_large model on a singe GPU like V100/A100(32G/40G) for caption task.
I found that the max batch size is 2/4 on V100/A100. Is there any ways to increase the batch size? fp16 is already set

**Comment by JustinLin610** [comment]
Try gradient accumulation with `--update-freq`

**Comment by dinglei8908** [comment]
> Try gradient accumulation with `--update-freq`

 thanks for suggestions. max mini batch size is 2/4 on V100/A100 for caption task, does this make sense?
